cmd/vic-machine/common/proxy.go

```diff
@@ -44,13 +47,20 @@ func (p *Proxies) ProxyFlags() []cli.Flag {
 			Usage:  "An HTTP proxy for use when fetching images, in the form http(s)://fqdn_or_ip:port",
 			Hidden: true,
 		},
+		cli.GenericFlag{
+			Name:   "no-proxy",
+			Value:  flags.NewOptionalString(&p.NoProxy),
+			Usage:  "URLs that should be excluded from proxying. This should be a comma-separated list of hostnames, domain names, or a mixture of both",
```

Comment:
In the example below, you have ".example.com". Presumably this means example.com and all sub-domains. It's not clear from this help output that this would be accepted. Perhaps @stuclem can suggest a concise way to explain this.